src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -650,43 +657,44 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
             graph_id.erase(std::remove(graph_id.begin(), graph_id.end(), '\r'),
                            graph_id.end());
 
-            if (!JasmineGraphFrontEnd::graphExistsByID(graph_id,sqlite)) {
+            if (!JasmineGraphFrontEnd::graphExistsByID(graph_id, sqlite)) {
                 string error_message = "The specified graph id does not exist";
                 result_wr = write(connFd, error_message.c_str(), FRONTEND_COMMAND_LENGTH);
 
-                if(result_wr < 0) {
+                if (result_wr < 0) {
                     frontend_logger.log("Error writing to socket", "error");
                 }
 
                 result_wr = write(connFd, "\r\n", 2);
 
-                if(result_wr < 0) {
+                if (result_wr < 0) {
                     frontend_logger.log("Error writing to socket", "error");
                 }
             } else {
                 auto begin = chrono::high_resolution_clock::now();
-                long triangleCount = JasmineGraphFrontEnd::countTriangles(graph_id,sqlite,masterIP);
+                long triangleCount = JasmineGraphFrontEnd::countTriangles(graph_id, sqlite, masterIP);
                 auto end = chrono::high_resolution_clock::now();
                 auto dur = end - begin;
                 auto msDuration = std::chrono::duration_cast<std::chrono::milliseconds>(dur).count();
-                frontend_logger.log("Triangle Count: " + to_string(triangleCount) + " Time Taken: " + to_string(msDuration) +
-                " milliseconds", "info");
-                result_wr = write(connFd, to_string(triangleCount).c_str(), (int)to_string(triangleCount).length());
-                if(result_wr < 0) {
+                frontend_logger.log(
```

Comment:
Why do we change the triangle counting related code here?

Reply:
The change is from code formatting which has happened to the whole class.